applications/StructuralMechanicsApplication/tests/test_compute_moment_of_inertia.py

```diff
@@ -0,0 +1,229 @@
+
+    def _create_shell_nodes(self,mp):
+        mp.CreateNewNode(1,0.0,0.0,0.0)
+        mp.CreateNewNode(2,0.2,0.0,0.0)
+        mp.CreateNewNode(3,0.2,0.1,0.0)
+        mp.CreateNewNode(4,0.0,0.1,0.0)
+        mp.CreateNewNode(5,0.6,0.0,0.0)
+        mp.CreateNewNode(6,0.6,0.1,0.0)
+        mp.CreateNewNode(7,1.0,0.0,0.0)
+        mp.CreateNewNode(8,1.0,0.1,0.0)
+        mp.CreateNewNode(9,0.0,0.4,0.0)
+        mp.CreateNewNode(10,0.2,0.4,0.0)
+        mp.CreateNewNode(11,0.6,0.4,0.0)
+        mp.CreateNewNode(12,1.0,0.4,0.0)
+        mp.CreateNewNode(13,1.0,0.5,0.0)
+        mp.CreateNewNode(14,0.6,0.5,0.0)
+        mp.CreateNewNode(15,0.2,0.5,0.0)
+        mp.CreateNewNode(16,0.0,0.5,0.0)
+
+    def _create_shell_elements(self,mp,element_name = "ShellThinElementCorotational3D4N"):
+        mp.CreateNewElement(element_name, 1, [1,2,3,4], mp.GetProperties()[1])
+        mp.CreateNewElement(element_name, 2, [2,5,6,3], mp.GetProperties()[1])
+        mp.CreateNewElement(element_name, 3, [5,7,8,6], mp.GetProperties()[1])
+        mp.CreateNewElement(element_name, 4, [4,3,10,9], mp.GetProperties()[1])
+        mp.CreateNewElement(element_name, 5, [3,6,11,10], mp.GetProperties()[1])
+        mp.CreateNewElement(element_name, 6, [6,8,12,11], mp.GetProperties()[1])
+        mp.CreateNewElement(element_name, 7, [9,10,15,16], mp.GetProperties()[1])
+        mp.CreateNewElement(element_name, 8, [10,11,14,15], mp.GetProperties()[1])
+        mp.CreateNewElement(element_name, 9, [11,12,13,14], mp.GetProperties()[1])
+
+    def test_nodal_moi(self):
+        dim = 3
+        nr_nodes = 4
+        current_model = KratosMultiphysics.Model()
+        mp = current_model.CreateModelPart("structural_part_nodal_masses")
+        mp.ProcessInfo[KratosMultiphysics.DOMAIN_SIZE] = dim
+
+        # create nodes
+        dx = 1.2
+        for i in range(nr_nodes):
+            mp.CreateNewNode(i+1,i*dx,0.00,0.00)
+
+        # create elements
+        elem1 = mp.CreateNewElement("NodalConcentratedElement2D1N", 1, [1], mp.GetProperties()[0])
+        elem2 = mp.CreateNewElement("NodalConcentratedElement2D1N", 2, [2], mp.GetProperties()[0])
+        elem3 = mp.CreateNewElement("NodalConcentratedElement3D1N", 3, [3], mp.GetProperties()[0])
+        elem4 = mp.CreateNewElement("NodalConcentratedElement3D1N", 4, [4], mp.GetProperties()[0])
+
+        elem1.SetValue(KratosMultiphysics.NODAL_MASS,21.234)
+        elem2.SetValue(KratosMultiphysics.NODAL_MASS,5.234)
+        elem3.SetValue(KratosMultiphysics.NODAL_MASS,112.234)
+        elem4.SetValue(KratosMultiphysics.NODAL_MASS,78.234)
+
+        p1 = KratosMultiphysics.Point(1.8, 0.0, 0.0)
+        p2 = KratosMultiphysics.Point(1.8, 2.0, 0.0)
+
+        moi_process = StructuralMechanicsApplication.ComputeMomentOfInertiaProcess(mp, p1, p2)
+        moi_process.Execute()
+        moment_of_inertia = mp.ProcessInfo[StructuralMechanicsApplication.MOMENT_OF_INERTIA]
```

Comment:
Process info?

Reply:
Why the inertia moment is saved in the process info?

Reply:
just following what you did for the MassProcess:
https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/custom_processes/total_structural_mass_process.cpp#L112